recipe/meta.yaml

```diff
@@ -20,38 +20,43 @@ build:
 requirements:
   host:
     - git
-    - python >=3.8
+    - python >=3.8,<3.11
     - pip !=21.3
     - pytest-runner
     - setuptools_scm
   run:
     # esmvaltool
-    - python >=3.8
-    - iris >=3.2.1
+    - python >=3.8,<3.11
+    - iris >=3.4.0
     - python-stratify
     - scipy >=1.6
     # Normally installed via pip:
     - cartopy
     - cftime
-    - cf-units >=3.0.0, <3.1.0, !=3.0.1.post0
+    - cf-units
+    - compilers
```

Comment:
Why are compilers needed?

Reply:
I had the same question. In #57, this works without compilers, so I am in favor of leaving it out.

Reply:
@bouweandela mentioned on Slack that `compilers` was needed for `vprof` that we no longer require, so we should be good without it, bud :+1: